### PR TITLE
feat(frontend): [NO-ISSUE] translations for Sign up form

### PR DESCRIPTION
**Pull request by mateusz-kurowskiUG**

### CODE REVIEW

File: frontend/src/app/[locale]/auth/_components/SignUpForm.tsx [hunk 1]
```diff
@@ -1,7 +1,112 @@
...
-	return <div>SignUpForm</div>;
+	const t = useTranslations("SignUpForm");
+	// TODO: Add password regex
+	const formSchema = z
+		.object({
+			email: z
+				.string()
+				.min(5, {
+					message: t("emailTooShortError"),
+				})
```
Comment (Memetelve) [inline frontend/src/app/[locale]/auth/_components/SignUpForm.tsx:28]: Check redundant. Any valid email cannot be shorter that 5 chars

File: frontend/messages/pl.json [hunk 2]
```diff
@@ -2,5 +2,19 @@
...
+		"passwordTooShortError": "Zbyt krótkie hasło.",
+		"passwordTooLongError": "Hasło jest zbyt długie.",
+		"noPasswordMatchError": "Wprowadzone hasła nie zgadzają się.",
+		"registerButtonText": "Zarejestruj się",
+		"emailInputLabel": "Twoj email",
+		"passwordInputLabel": "Hasło",
+		"confirmPasswordInputLabel": "Hasło",
+		"emailPlaceHolder": "Email",
+		"passwordPlaceHolder": "Hasło",
+		"confirmPasswordPlaceHolder": "Hasło"
```
Comment (olszewskista) [inline frontend/messages/pl.json:18]: we have different properties with the same repeated values. should we use one property in multiple places?

File: frontend/src/app/[locale]/auth/_components/SignUpForm.tsx (hunk 1 above)
Comment (mateusz-kurowskiUG) [reply frontend/src/app/[locale]/auth/_components/SignUpForm.tsx:28]: temporary, until regex is done

File: frontend/messages/pl.json (hunk 2 above)
Comment (mateusz-kurowskiUG) [reply frontend/messages/pl.json:18]: no.

Comment (olszewskista) [reply frontend/messages/pl.json:18]: aha

File: frontend/src/app/[locale]/auth/_components/SignUpForm.tsx (hunk 1 above)
Comment (mateusz-kurowskiUG) [reply frontend/src/app/[locale]/auth/_components/SignUpForm.tsx:28]: feat(frontend): [PYTA-16] add signup page *template*

File: frontend/messages/pl.json (hunk 2 above)
Comment (mateusz-kurowskiUG) [reply frontend/messages/pl.json:18]: This is going to be changed, but something needs to be there temporarily

File: frontend/src/app/[locale]/auth/_components/SignUpForm.tsx (hunk 1 above)
Comment (Memetelve) [reply frontend/src/app/[locale]/auth/_components/SignUpForm.tsx:28]: yes, but why add something that should never be there, also [https://emailregex.com/](https://emailregex.com/)

**Pull request closed without merging**